recipes/tgbot/all/conanfile.py

```diff
@@ -77,4 +88,5 @@ def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
 
     def package_info(self):
-        self.cpp_info.libs = ['TgBot']
+        self.cpp_info.libs = ["TgBot"]
+        self.cpp_info.requires = ["boost::headers", "boost::system", "libcurl::libcurl", "openssl::openssl"]
```

Comment:
vs this

```suggestion
        self.cpp_info.requires = ["boost::system", "libcurl::libcurl", "openssl::openssl"]
```

Reply:
boost headers are always available, there is no `without_headers` option in boost recipe.

Reply:
Doesn't system require headers?

Reply:
Sure, but I don't know if tgbot depends on system only, or some other headers-only lib, and I prefer to be explicit on direct dependencies.